bin/refresh-verify-change.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+#/ Builds and verifies the specified plan and any reverse dependencies
+#/
+#/ Usage:
+#/
+#/ refresh-verify-change.sh NAME
+#/
+#/ ex: refresh-verify-change.sh glibc
+
+set -euo pipefail
+
+export HAB_NONINTERACTIVE=true
+export HAB_NOCOLORING=true
+
+usage() {
+  grep '^#/' < "${0}" | cut -c4-
+  exit 1
+}
+
+need_cmd() {
+  cmd="$1"
+
+  if ! command -v "$cmd" >/dev/null; then
+    echo "Could not find $cmd; Please install it first"
+  fi
+}
+
+[[ -z "${1:-}" ]] && usage
+
+changed_plan="$1"
+[[ -d "$changed_plan" ]] || usage
+
+need_cmd "curl"
+need_cmd "jq"
+need_cmd "tee"
+
+rdeps_url="https://bldr.habitat.sh/v1/rdeps/core/$changed_plan/group"
+query='.rdeps | .[] | .idents | map(select(startswith("core"))) |select(length > 0) | .[] | ltrimstr("core/")'
+build_id="$(date -u +%Y%m%d%H%M%S)"
+build_log_dir="results/$changed_plan-$build_id"
+
+echo "Setting build_id: ${build_id}"
+mkdir -p "$build_log_dir"
+
+echo "Getting reverse dependency list"
+echo "$changed_plan" > "$build_log_dir"/build_order
+curl -s "$rdeps_url" | jq -r "$query" >> "$build_log_dir"/build_order
+
+echo "Building $changed_plan and reverse_dependencies"
+while read -r plan; do
+  if [[ "$plan" == hab-* || "$plan" == builder-* ]];  then
+    echo "Skipping $plan."
+    continue
+  fi
+  if [[ -n "${BASE_ONLY:-}" ]] && ! grep -q "$plan" base-plans.txt; then
```

Comment:
I don't think [base-plans.txt](https://github.com/habitat-sh/core-plans/blob/master/base-plans.txt) has any "major version" plans; but potentially this grep might match `foobar` against a base plan entry `foobar8`.

Reply:
I think there's other cases where one plan name is a substring of another plan.

Reply:
I think there's other cases where one plan name is a substring of another plan.

Reply:
Good point, I'll fix this